backend/src/main/resources/application.yaml

```diff
@@ -49,6 +49,12 @@ spring:
     jdbc:
       initialize-schema: never
       table-name: ${spring.jpa.properties.hibernate.default_schema}.spring_session
+  servlet:
+    multipart:
+      max-file-size: 50MB
```

Comment:
This is set to the same max file limit enforced by the frontend for result upload